src/styles/navbar.scss

```diff
@@ -0,0 +1,119 @@
+$primaryColor: #075df2;
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+nav i {
+  color: blue;
+  font-size: 100px;
+}
+li {
+  list-style-type: none;
+}
+a {
+  text-decoration: none;
+}
+nav {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+}
+nav div {
+  font-size: 1rem;
+  font-weight: 500;
+}
+.ul-links {
+  display: flex;
+  justify-content: space-around !important;
```

Comment:
Remove all of the `!important`; it should be only used to override other styles that we don't have control over. This is not a good practice